### DISCUSSION TITLE
error 500 internal server error after migration

### DISCUSSION THREAD

**Discussion by lcf-sbh** [grafana/grafana-infinity-datasource#451] [Q&A] [answered]
Hi!
Thank you Yesoreyam for the awesome grafana plug-in.

I have been using infinity v1.0.0 for a few months on a local ubuntu grafana server.
I use the infinity data-source to get json data from REST APIs.

I recently moved to a google cloud VM running debian.
At the same time, I also upgraded to infinity 1.1.0.

Ever since I migrated, I wasn't able to get data from one of my APIs.
I get "error getting data frame. 500 Internal Server Error".

I can still reach said API with postman.
I can reach other APIs without problem.

Here is the query inspector :
![image](https://user-images.githubusercontent.com/117833718/200847388-d533a3d7-e5b5-4de9-b97a-0824156b3075.png)

I am pretty sure I missed something in the new configuration.

Where does the argument "q=0.9,text/plain" comes from?
Can it be the cause for the 500 internal server error?
If that's not the cause, what can it be?

thanks,
lcf-sbh.

**Comment by yesoreyeram** [comment, marked as answer]
`q=0.9,text/plain` as a fallback. But not sure if your target API finds it irrelevant and throws 500. If that is the case, add custom header "Accept" and pass the values as you need.

if you are in doubt, remove that part form curl command and run outside grafana to validate.

**Reply by lcf-sbh** [reply]
thanks a lot for your response.

it does work in curl from my server without the 'q=0.9,text/plain'.

![image](https://user-images.githubusercontent.com/117833718/200855323-9ede2cdd-a9bb-49c2-b8f9-f5e98277c288.png)

How can I get it to work in grafana?

**Reply by lcf-sbh** [reply]
Yup, soory, you already answered.

and it works 👍 
![image](https://user-images.githubusercontent.com/117833718/200856305-87c7d647-6174-44bc-b183-61435ed6c7df.png)

Thanks a million!
